samples/openapi3/client/petstore/dart2/petstore_client_lib_fake/lib/model/enum_arrays.dart

```diff
@@ -250,7 +250,7 @@ class EnumArraysArrayEnumEnumTypeTransformer {
   /// and users are still using an old app with the old code.
   EnumArraysArrayEnumEnum? decode(dynamic data, {bool allowNull = true}) {
```

Comment:
wouldn't this imply that `data` is no longer `dynamic` type?

Reply:
it's dynamic, since it's either string or num